test/serializers/serializer_for_with_namespace_test.rb

```diff
@@ -0,0 +1,85 @@
+require 'test_helper'
+
+module ActiveModel
```

Comment:
looks good. needs controller test for implicit lookup as well as option passing, which is really covered below, here